deepsea.spec.in

```diff
@@ -491,7 +491,7 @@ systemctl try-restart salt-api > /dev/null 2>&1 || :
 %config /srv/salt/ceph/configuration/files/*.conf
```

Comment:
hmmm, aren't these the files users use to customize their ceph.conf? And the lack of `noreplace` here means deepsea is clobbering these on every update?

Reply:
nevermind - opening a PR